src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineJavaCheck.java

```diff
@@ -32,7 +32,8 @@
  * </p>
  * <p>
  * This class is variation on
- * <a href="https://checkstyle.org/config_regexp.html#RegexpSingleline">RegexpSingleline</a>
+ * <a href="https://checkstyle.org/checks/regexp/regexpsingleline.html#RegexpSingleline">
```

Comment:
Please remove `#RegexpSingleline`

Same for others.

Reply:
Done.